README.md

```diff
@@ -19,17 +19,19 @@ This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-opti
 
 ## Docker
 
+Don't change `.env.production` file and fill in `.env` file with real `NEXT_PUBLIC_API_URL` and `NEXT_PUBLIC_CHAIN_ID` variables.
```

Comment:
But how do you alternate between different chain ids? The idea is that you pass it in the container

```
docker run -p 3000:3000 -e HOSTNAME='0.0.0.0' --env-file .env expolore-stage-arbitrum
```



@at31oldfox is working on solution with proxy server for env vars btw...

Reply:
> But how do you alternate between different chain ids? The idea is that you pass it in the container
> 
> ```
> docker run -p 3000:3000 -e HOSTNAME='0.0.0.0' --env-file .env expolore-stage-arbitrum
> ```
> 
> @at31oldfox is working on solution with proxy server for env vars btw...

my fault. It would work with variables passed in the console as well. Updated readme